src/components/Cards/index.js

```diff
@@ -0,0 +1,29 @@
+import Card from './Card';
+import CardTitle from './CardTitle';
+import CardBody from './CardBody';
+import CardGrid from './CardGrid';
+import CardHeading from './CardHeading';
+import CardFooter from './CardFooter';
+import CardLink from './CardLink';
+import CardDropdownButton from './CardDropdownButton';
+import CardHeightMatching from './CardHeightMatching';
```

Comment:
can we also do a simple:
```
Card.Title = CardTitle;
Card.Body = CardBody;
Card.Heading = CardHeading;
Card.Footer = CardFooter;
Card.Link = CardLink;
Card.DropdownButton = CardDropdownButton;
Card.HeightMatching = CardHeightMatching;
```
like other subclassed components in this project ;)

This gives us really nice intellisense downstream...